packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -1,260 +0,0 @@
-import Ember from 'ember-metal/core';
-import run from 'ember-metal/run_loop';
-import Namespace from 'ember-runtime/system/namespace';
-import EmberView from 'ember-views/views/view';
-import Component from 'ember-views/components/component';
-import compile from 'ember-template-compiler/system/compile';
-import { A as emberA } from 'ember-runtime/system/native_array';
-
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-
-import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
-import viewKeyword from 'ember-htmlbars/keywords/view';
-import ComponentLookup from 'ember-views/component_lookup';
-import jQuery from 'ember-views/system/jquery';
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var originalLookup = Ember.lookup;
-
-var view, lookup, owner, TemplateTests, originalViewKeyword;
-
-QUnit.module('ember-htmlbars: {{#if}} and {{#unless}} helpers', {
-  setup() {
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-
-    Ember.lookup = lookup = {};
-    lookup.TemplateTests = TemplateTests = Namespace.create();
-    owner = buildOwner();
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.registerOptionsForType('view', { singleton: false });
-    owner.registerOptionsForType('component', { singleton: false });
-    owner.register('view:toplevel', EmberView.extend());
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-
-    Ember.lookup = lookup = originalLookup;
-    TemplateTests = null;
-
-    resetKeyword('view', originalViewKeyword);
-  }
-});
-
-QUnit.test('properties within an if statement should not fail on re-render', function() {
```

Comment:
deleted as this is covered by `Syntax test: {{#if}} and {{#unless}} without inverse` tests